cocos/core/data/editor-extendable.ts

```diff
@@ -23,7 +23,7 @@ class Empty {}
 
 /**
  * Class which implements the `EditorExtendableObject` interface.
- * @internal DO NOT USE IT
+ * @engineInternal
  */
 export const EditorExtendable = editorExtendableInternal();
```

Comment:
As i remembered, you corrected my modification from `@engineInternal` to current implementation, or editor will have error.

Reply:
because we haven't supported move engineInternal interface into __private namespace.
so Editor can't reference the base class EditorExtendable. Now we've fixed it